charts/datadog/templates/_container-security-agent.yaml

```diff
@@ -91,9 +91,11 @@
     - name: cgroups
       mountPath: /host/sys/fs/cgroup
       readOnly: true
+    {{- if not .Values.datadog.processAgent.disablePasswdMount }}
```

Comment:
since the option is used in my agent a not only for the processAgent, can we move the option outside the `.Values.datadog.processAgent` options section?

Reply:
updated :)